src/utilities/theme/role-variants.ts

```diff
@@ -186,8 +186,8 @@ export const roleVariants: RoleVariants = {
     {
       name: 'interactiveAction',
       description: 'Used for links and plain buttons.',
-      light: {lightness: 44},
-      dark: {lightness: 65},
+      light: {lightness: 48.6},
+      dark: {lightness: 65, saturation: 100, hue: 247.6},
```

Comment:
This change, and the changes to `interactiveActionDisabled`, `interactiveActionHovered`, and  `interactiveActionPressed` mean that the interactive action color is no longer configurable. Is that the intention?

Reply:
I found out that with blue, the Hue is treated differently than it is in other colour spaces. When we make it lighter, it also shifts to be more purple. 

I can try starting it from a "bluer" starting place to see if we can compensate, but if not we may need to lock it down in this way. 

I figured this was ok for our internal needs, and that we could ship a cleaner version for partners.

Reply:
Finding a way for this to work without fixing the hue is ultimately what we'll need to figure out. Can you find a more blue starting place that looks good? Otherwise this could be handled universally as a relative shift of `-7.1`, or we could get more complicated and only do shifts based on hue, or have a specific role variant config for our design language which is different than the factory default.

For now we can ship and think on it async unless there is a great answer now.